### ISSUE TITLE
[Bug]: Missing docker image for ARM

### ISSUE THREAD

**Issue by iocron** [toniblyx/prowler#4156]
### Steps to Reproduce

```
docker pull toniblyx/prowler
Using default tag: latest
latest: Pulling from toniblyx/prowler
no matching manifest for linux/arm64/v8 in the manifest list entries
```

### Expected behavior

Should pull a existing docker image for ARM, which does not exist as of right now => Solution: Can be build by a github workflow. 

### Actual Result with Screenshots or Logs

- 

### How did you install Prowler?

Docker (docker pull toniblyx/prowler)

### Environment Resource

MacOS 14.4
Chip: Apple M1 Pro
Docker Engine: 25.0.3

### OS used

MacOS 14.4

### Prowler version

latest

### Pip version

24.0

**Comment by jfagoagas** [comment]
Hi @iocron we are not builiding the container images from ARM yet. As a workaround you can use the following command to get the Prowler image `docker pull --platform linux/amd64 toniblyx/prowler`.

Please let me know if this works for you.

Thanks for using Prowler 🚀

**Comment by jfagoagas** [comment]
Hi @iocron I'll close this issue. Please feel free to reopen it or reach to us if this doesn't work for you.

Thanks for using Prowler 🚀

**Comment by pcshikhar** [comment]
> Hi @iocron we are not builiding the container images from ARM yet. As a workaround you can use the following command to get the Prowler image `docker pull --platform linux/amd64 toniblyx/prowler`.
> 
> Please let me know if this works for you.
> 
> Thanks for using Prowler 🚀

It would help if you add this to the doc site here - https://docs.prowler.com/projects/prowler-open-source/en/latest/#__tabbed_2_3

**Comment by jfagoagas** [comment]
Hello @pcshikhar we've just add that to our documentation, thank you for the hint!

**Comment by piljaechae** [comment]
@jfagoagas ARM is the mainstream now, how about supporting it?

**Comment by jfagoagas** [comment]
Hello @piljaechae I apologise for the late response. We've just merged #8773 which builds our container images for `arm64`.

Thanks for using Prowler 🚀
